assets/auto-vj-uri.html

```diff
@@ -27,38 +35,45 @@
 
             function tryCreatePlayer(meta) {
                 var vLayer = document.createElement('video');
+                vLayer.setAttribute("id", meta.deck);
                 vLayer.muted = true;
                 vLayer.autoplay = false;
                 vLayer.defaultPlaybackRate = meta.tempo;
                 vLayer.playbackRate = meta.tempo;
-                vLayer.addEventListener('loadeddata', () => {
-                    if(vLayer.readyState == HTMLMediaElement.HAVE_ENOUGH_DATA && players[meta.filePath] && !vLayer.didStart) { // can play 100% and not removed yet
-                        document.getElementById('container').appendChild(vLayer);
-                        vLayer.play();
-                        vLayer.didStart = true;
-                    }
-                });
                 vLayer.addEventListener('ended',() => {
-                    vLayer.style.display = "none";
+                    // Hide video player if it reaches the end to avoid black screen
+                    $(vLayer).fadeOut('fast');
```

Comment:
If this is the only use of the effect, how about using the CSS animation and opacity properties on the video element? And adding/removing a `.hidden` class for instance.